lib/tasks/eu_exit_business_finder.rake

```diff
@@ -0,0 +1,155 @@
+          "/guidance/the-farming-sector-and-preparing-for-eu-exit",
+          "/government/publications/upholding-environmental-standards-if-theres-no-brexit-deal"
+        ],
+        "old_facet_value": "94b3cfe2-af89-4744-b8d7-7fc79edcbc85",
+        "new_facet_value": "9d54c591-f5ca-4d0c-a484-12d5591987cb"
+      },
+      {
+        "base_paths": [
+          "/government/publications/approval-and-operation-of-plant-health-inspection-facilities-at-place-of-first-arrival",
+          "/guidance/importing-and-exporting-plants-and-plant-products-if-theres-no-withdrawal-deal",
+          "/guidance/trading-timber-imports-and-exports-if-theres-no-brexit-deal"
+        ],
+        "old_facet_value": "94b3cfe2-af89-4744-b8d7-7fc79edcbc85",
+        "new_facet_value": "afd45eef-743a-417d-9245-3eab8322116d"
+      }
+    ]
+  end
+
+  def facet_values_to_untag
+    # For each item in the array:
+    # All content tagged to that facet value will be untagged
+    [
+      "7536c0c4-fb41-43f4-a2c4-08f4fa9f5427",
+      "5faa1741-fc55-4110-b342-de92f6324118",
+      "c1d8057c-76bf-431c-9ac8-6281a4b7b9ca",
+      "14cf2a68-3297-44d3-ba01-a4426845b1b8",
+      "040649fc-4e2c-4028-b846-77fe3eebd1f7",
+      "94b3cfe2-af89-4744-b8d7-7fc79edcbc85"
+    ]
+  end
+
+  def replace_facet_values
+    facet_values_merges = facet_values_to_replace
+
+    content_changes = []
+    facet_values_merges.each do |facet_values_merge|
+      content_ids = []
+      content_links = Services.rummager.search_enum(
+        filter_facet_values: facet_values_merge[:old_facet_value]
+      ).pluck("link")
+      content_links.each do |content_link|
+        content_ids << Services.rummager.get_content(content_link).to_hash['raw_source']['content_id']
+      end
+      content_changes << {
+        "content_ids": content_ids,
+        "old_facet_value": facet_values_merge[:old_facet_value],
+        "new_facet_value": facet_values_merge[:new_facet_value]
+      }
+    end
+
+    bulk_retag_documents(content_changes)
+  end
+
+  def retag_content_facet_values
+    content_changes = content_to_update_facet_values
+
+    content_changes.each do |content_change|
+      content_change['content_ids'] = Services.publishing_api.lookup_content_ids(base_paths: content_change[:base_paths]).values
+    end
+
+    bulk_retag_documents(content_changes)
+  end
+
+  def delete_facet_values_from_content
+    facet_values = facet_values_to_untag
+
+    facet_values.each do |facet_value|
+      content_ids = get_content_from_facet_value(facet_value)
+      content_ids.each do |content_id|
+        links = Services.publishing_api.get_links(content_id).to_hash.fetch('links', {})
+        content_facet_values = links.fetch('facet_values', [])
+        content_facet_values.delete(facet_value)
+        links['facet_values'] = content_facet_values
+        Services.publishing_api.patch_links(content_id, links: links)
+      end
+    end
+  end
+
+  def get_content_ids_from_facet_value(facet_value)
```

Comment:
It looks like this method is not being used.

Reply:
I think it's supposed to be called `get_content_from_facet_value`

Reply:
This is something a test would have picked up 😃.

If the intention is to keep this rake task around for future generations, a test might be a good idea.